src/plugins/interactive_setup/public/use_form.ts

```diff
@@ -153,6 +154,7 @@ export function useFormState<Values extends FormValues, Result>({
     const nextErrors = await validateForm(formValues);
     setTouched(mapDeep(formValues, true));
     setSubmitCount(submitCount + 1);
+    // @ts-expect-error
```

Comment:
Nit: I personally prefer to add some text to this comment explaining what we're trying to ignore. At a minimum, I just copy the TS error verbatim if it's not too long